app/controllers/users_controller.rb

```diff
@@ -0,0 +1,3 @@
+class UsersController < ApplicationController
```

Comment:
- [ ] Hey there, awesome job on setting up the users controller! 🌟 However, we noticed a small deviation from the project requirements. As per the specifications, the users controller should include both the #index and #show actions. Currently, only the #index action is implemented. 🤔 To align with the project requirements, please ensure to add the corresponding view for the #show action, and don't forget to update the routes to include a route for the users' show page. 🚀 Let's make sure we cover all the bases and keep up the great work! 👏💻

```ruby
# app/controllers/users_controller.rb
class UsersController < ApplicationController
  
  def index; end

  def show; end

end

# config/routes.rb

  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end



```

Reply:
Thanks @zeff96 let me look into it